requirements.txt

```diff
@@ -1,2 +1,3 @@
 pbr<1.0.0
 python-openstacksdk
+os_client_config
```

Comment:
I guess the package name should be os-client-config?
Also please sort the package names.

Reply:
Both work. Will modify it